build-parent/pom.xml

```diff
@@ -27,7 +27,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
-    <quarkus.platform.version>2.3.0.Final</quarkus.platform.version>
+    <quarkus.platform.version>2.4.2.Final</quarkus.platform.version>
```

Comment:
Apparently we were hit by https://github.com/quarkusio/quarkus/issues/19494 . After upgrading to 2.4.2, I've got 3 green CI checks on a row. I'd say it fixed my issues with CI @ricardozanini @sutaakar